Trivago/Jade/Application/JsonApi/Request/RequestFactory.php

```diff
@@ -283,6 +298,20 @@ private function parseFilters(ResourceConfig $resourceConfig, CompositeFilter $c
         }
     }
 
+    /**
+     * Validating the "path" is not under "excluded_attributes" config section
+     * @param ResourceConfig $resourceConfig
+     * @param $key
```

Comment:
Please add the types for `$key` and `$path` in the PHPdoc block :)

Reply:
OK, no problem.